mingw-w64-qemu/PKGBUILD

```
@@ -1,94 +1,168 @@
-# Maintainer: Martell Malone <martellmalone@gmail.com>
-
 _realname=qemu
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
 _base_ver=5.2.0
 _rc=
 pkgver=${_base_ver}${_rc//-/.}
 pkgrel=1
-pkgdesc="A generic and open source processor emulator (mingw-w64)"
+pkgdesc="QEMU - a generic and open source machine emulator and virtualizer (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64')
 license=('GPL2' 'LGPL2')
 url="https://qemu.org/"
-makedepends=("${MINGW_PACKAGE_PREFIX}-gcc"
-             "${MINGW_PACKAGE_PREFIX}-python"
-             'perl' 'bison' 'bsdtar')
-depends=("${MINGW_PACKAGE_PREFIX}-capstone"
-         "${MINGW_PACKAGE_PREFIX}-curl"
-         "${MINGW_PACKAGE_PREFIX}-cyrus-sasl"
-         "${MINGW_PACKAGE_PREFIX}-glib2"
-         "${MINGW_PACKAGE_PREFIX}-gnutls"
-         "${MINGW_PACKAGE_PREFIX}-gtk3"
-         "${MINGW_PACKAGE_PREFIX}-libjpeg"
-         "${MINGW_PACKAGE_PREFIX}-libpng"
-         "${MINGW_PACKAGE_PREFIX}-libssh"
-         "${MINGW_PACKAGE_PREFIX}-libtasn1"
-         "${MINGW_PACKAGE_PREFIX}-libusb"
-         "${MINGW_PACKAGE_PREFIX}-libxml2"
-         "${MINGW_PACKAGE_PREFIX}-lzo2"
-         "${MINGW_PACKAGE_PREFIX}-nettle"
-         "${MINGW_PACKAGE_PREFIX}-pixman"
-         "${MINGW_PACKAGE_PREFIX}-snappy"
-         "${MINGW_PACKAGE_PREFIX}-SDL2"
-         "${MINGW_PACKAGE_PREFIX}-SDL2_image"
-         "${MINGW_PACKAGE_PREFIX}-libslirp"
-         "${MINGW_PACKAGE_PREFIX}-usbredir"
-         "${MINGW_PACKAGE_PREFIX}-zstd")
+# optional features which introduce many dependencies should be added to makedepends
+# for activation and optdepends for information
+makedepends=(
+  "${MINGW_PACKAGE_PREFIX}-gcc"
+  "${MINGW_PACKAGE_PREFIX}-ninja"
+  "${MINGW_PACKAGE_PREFIX}-python"
+  "${MINGW_PACKAGE_PREFIX}-python-sphinx"
+  "${MINGW_PACKAGE_PREFIX}-spice"
+  'perl' 'bison' 'bsdtar')
+depends=(
+  "${MINGW_PACKAGE_PREFIX}-capstone"
+  "${MINGW_PACKAGE_PREFIX}-curl"
+  "${MINGW_PACKAGE_PREFIX}-cyrus-sasl"
+  "${MINGW_PACKAGE_PREFIX}-glib2"
+  "${MINGW_PACKAGE_PREFIX}-gnutls"
+  "${MINGW_PACKAGE_PREFIX}-gtk3"
+  "${MINGW_PACKAGE_PREFIX}-libjpeg"
+  "${MINGW_PACKAGE_PREFIX}-libnfs"
+  "${MINGW_PACKAGE_PREFIX}-libpng"
+  "${MINGW_PACKAGE_PREFIX}-libslirp"
+  "${MINGW_PACKAGE_PREFIX}-libssh"
```

For clang when dependencies are eventually available.

```suggestion
  "${MINGW_PACKAGE_PREFIX}-libssh"
  "${MINGW_PACKAGE_PREFIX}-libssp"
```

---

Yeah, I search for it while preparing the recipe, but https://packages.msys2.org/base/mingw-w64-libssp shows only clang packages. Won't this break the other builds?

---

it's provided by mingw-w64-gcc-libs too to prevent that (and it's the truth: it does in fact also provide libssp)

---

Will test right now